README.md

```diff
@@ -189,6 +190,45 @@ class TritonPythonModel:
     that is created must have "TritonPythonModel" as the class name.
     """
 
+    def auto_complete_config(self):
```

Comment:
This should accept an additional argument named `model_config` that the user will use to auto_complete.

Reply:
I was wondering that since we put all the auto-complete logic and checking in c++, do we still need to pass the `model_config` argument to this function?

Reply:
They still need to receive the original model configuration (if any) that was provided by the server or config.pbtxt file and that is why I think they would still need to access the original model configuration.